### PR TITLE
Feature/mds 2977 display who edited conditions

### PR DESCRIPTION
**Pull request by mhuseinov**
Displays who and when last edited Conditions on a permit.
image

### CODE REVIEW

File: services/core-api/app/api/mines/permits/permit_conditions/models/permit_conditions.py [hunk 1]
```diff
@@ -46,6 +47,15 @@ class _ModelSchema(Base._ModelSchema):
         order_by='asc(PermitConditions.display_order)',
         backref=backref('parent', remote_side=[permit_condition_id]))
 
+    update_user = db.Column(
```
Comment (mmumby) [inline services/core-api/app/api/mines/permits/permit_conditions/models/permit_conditions.py:50]: I don't believe we want to expose the Audit columns to the FE. 

Can you do something similar to `last_updated_by` and `last_updated_date` on `now_application` ?

Comment (mmumby) [reply services/core-api/app/api/mines/permits/permit_conditions/models/permit_conditions.py:50]: @nayr974 @JustinMacaulay I might be wrong about the audit columns, can you guys confirm?

Comment (exzizt) [reply services/core-api/app/api/mines/permits/permit_conditions/models/permit_conditions.py:50]: > @nayr974 @JustinMacaulay I might be wrong about the audit columns, can you guys confirm?

That's correct. `update_user`, `update_timestamp`, `create_user`, and `create_timestamp` fields are to never be exposed

**Review by exzizt — APPROVED** [review]
Thank you for making those changes. Looks correct! 🥇

**Pull request merged**